### ISSUE TITLE
Update learning rate based on a momentum parameter.

### ISSUE THREAD

**Issue by Kaushalya** [Kaushalya/gnn-meta-attack-pytorch#3]
The current implementation uses the same learning rate in all iterations of the inner training loop.
